rust-stage0.yaml

```diff
@@ -1,14 +1,14 @@
 package:
   name: rust-stage0
-  version: 1.65.0
-  epoch: 2
+  version: 1.68.2
+  epoch: 0
   description: "bootstrap package for rust"
   copyright:
-    - license: blessing
+    - license: Apache-2.0 AND MIT
   dependencies:
     runtime:
     provides:
-      - rust=1.65.0
+      - rust=1.68.2
```

Comment:
Since `${{package.version}}` isn't supported here yet, can you add a comment up by the `version:` to update this `provides` when we do this in the future? I can see this being easy to miss.

Reply:
Done.